packages/api/README.md

```diff
@@ -29,11 +29,12 @@ See [Cumulus README](https://github.com/cumulus-nasa/cumulus/blob/master/README.
 Running tests for kinesis-consumer depends on localstack. Once you have installed localstack, you can start it for dynamoDB only:
 
 ```
-SERVICES=dynamodb localstack start
+LAMBDA_EXECUTOR=docker localstack start
 ```
 
 Then you can run tests locally via:
 
 ```bash
-LOCALSTACK_HOST=localhost npm run test
+export DOCKERHOST=$(ifconfig | grep -E "([0-9]{1,3}\.){3}[0-9]{1,3}" | grep -v 127.0.0.1 | awk '{ print $2 }' | cut -f2 -d: | head -n1)
```

Comment:
Looking at this, I don't think there is a way to run this locally with our main docker image. So I'm not sure anymore if we have to move this to the root level of the repo.

Reply:
I think the db-indexer tests need some work to get working from the root of the repo. I'm going to skip them for now in hopes of returning to get them running as part of the larger test suite. This could help testing other lambdas, especially those in @cumulus/api.